solutions/swb-reference/src/services/sshKeyService.test.ts

```diff
@@ -250,15 +251,112 @@ describe('SshKeyService', () => {
 
   describe('createSshKey', () => {
     let createSshKeyRequest: CreateSshKeyRequest;
+    let mockSshKeyId: string;
 
     beforeEach(() => {
-      createSshKeyRequest = { projectId: '', userId: '' };
+      mockSshKeyId = 'sshkey-mockvalues';
+      createSshKeyRequest = { projectId: 'proj-123', userId: '1234' };
+      sshKeyService['_getSshKeyId'] = jest.fn(() => mockSshKeyId);
```

Comment:
usually you don't want to be mocking anything on the object you're testing because it could mask problems in your class' logic. any way you can mock the service that is used to get the ssh key id instead?

Reply:
Got it. Yeah I figured out how to mock the underlying code.